docs/cql/functions.rst

```diff
@@ -228,6 +230,65 @@ A number of functions are provided to “convert” the native types into binary
 takes a 64-bit ``blob`` argument and converts it to a ``bigint`` value. For example, ``bigintAsBlob(3)`` is
 ``0x0000000000000003`` and ``blobAsBigint(0x0000000000000003)`` is ``3``.
 
+.. _udfs:
+
+User-defined functions
+~~~~~~~~~~~~~~~~~~~~~~
+
+User-defined functions (UDFs) execute user-provided code in ScyllaDB. Supported languages are currently Lua and WebAssembly.
+
+UDFs are part of the Cassandra schema, and are automatically propagated to all nodes in the cluster.
+UDFs can be overloaded, so that multiple UDFs with different argument types can have the same function name, for example::
+
+   CREATE FUNCTION sample ( arg int ) ...;
+   CREATE FUNCTION sample ( arg text ) ...;
+
+Arguments for functions can be literals or terms. Prepared statement placeholders can be used, too.
```

Comment:
I would add a few words to the beginning of this sentence like, "When calling a user-defined function, arguments can be ...". Because it took me a few seconds to understand that you moved from the function definition (in the previous paragraph) to calling it (in this sentence).